llm/codellama/README.md

```diff
@@ -1,17 +1,29 @@
-# Serving Your Private Code Llama-70B with API, Chat, and VSCode Access
+<!-- $REMOVE -->
+# Code Llama: Serve Your Private Code Model with API, Chat, and VSCode Access
+<!-- $END_REMOVE -->
+<!-- $UNCOMMENT# Code Llama: Serve Your Private Code Model -->
 
 [Code Llama](https://github.com/facebookresearch/codellama) is a code-specialized version of Llama 2 that was created by further training Llama 2 on its code-specific datasets, sampling more data from that same dataset for longer. On Jan 29th, 2024, Meta released the Code Llama 70B, the largest and best-performing model in the Code Llama family.
 
 The followings are the demos of Code Llama 70B hosted by SkyPilot Serve (aka SkyServe) (see more details about the setup in later sections):
 
-##### Connect to hosted Code Llama with Tabby as a coding assistant in VScode
-<img src="https://imgur.com/fguAmP0.gif" width="60%" />
+## Demos
+<figure>
+<center>
+<img src="https://imgur.com/fguAmP0.gif" width="60%" title="Coding Assistant: Connect to hosted Code Llama with Tabby in VScode" />
 
-##### Connect to hosted Code Llama with FastChat for chatting
-<img src="https://imgur.com/Dor1MoE.gif" width="60%" /> 
+<figcaptuion>Coding Assistant: Connect to hosted Code Llama with Tabby in VScode</figcaption>
+</figure>
+
+<figure>
+<center>
+<img src="https://imgur.com/Dor1MoE.gif" width="60%" title="Chat: Connect to hosted Code Llama with FastChat" />
+
+<figcaption>Chat: Connect to hosted Code Llama with FastChat</figcaption>
+</figure>
 
 ## References
-* [Llama-2 Example](../../llm/llama-2/)
+* [Llama-2 Example](https://github.com/skypilot-org/skypilot/tree/master/llm/llama-2/)
```

Comment:
Does this mean in all future `llm/**/*.md` that get added to the gallery, no relative links should appear?

If so, worth a comment in gallery index's rst?

Reply:
Good point! Added. Thanks!